source/pasta/brusselSproutGnocchi.rst

```diff
@@ -0,0 +1,48 @@
+Gnocchi with Brussel Sprouts, Garlic, and a boat-load of butter
+===============================================================
+
+Details
+-------
+
+* Number of servings: 6
+* Cook time: 20 minutes
+
+Ingredients
+-----------
+
+* 1 lb Brussel sprouts, trimmed and halved so they have a big flat face
+* 500g gnocchi (or any type of pasta)
+* At least 4 cloves of garlic (prefferably more), finely minced
+* The juice and zest of 1 lemon (zest less important than juice)
+* 6 tbsp butter
+* Chilli flakes
+
+Optional:
+
+* Parmesan
+* 1/2 tsp honey
+
+Equipment
+---------
+
+* A saucepan for cooking the gnocchi/pasta
+* The widest frying pan you have, prefferably with high sides
+* A strainer
+
+Steps
+-----
+
+1. Put salted water on to boil
+2. Put a decent covering of olive oil in the frying pan, and place the sprouts in it, flat-side down.
```

Comment:
"Coat the bottom of a frying pan with olive oil and add the sprouts flat-side down."